post-scan-actions/aws-python-email-notification/README.md

```diff
@@ -24,7 +24,6 @@ After a scan occurs, this example Lambda function sends out a notification Email
         - In the **Name** field, enter a name. Example: `FSS_SES_Send_Email_Policy`.
         - Click **Create policy**.
     8. Click the link to your policy to open its summary.
-    9. Take note of the **Policy ARN** near the top of the page.
```

Comment:
It's would be used if the user then use the CLI to create execution role in next step. Does that make sense to you?

Reply:
Ah, I see! So we need it for the next step, which is to use the CLI. 
In that case, no need to delete it. Thanks @carpusherw